### PR TITLE
Update default homepage content for Slate

### PR DESCRIPTION
**Pull request by stevepiercy**
Here's the result.

Screen Shot 2022-08-26 at 3 17 59 AM

### CODE REVIEW

**Comment by stevepiercy** [comment]
@jenkins-plone-org please run jobs

**Comment by stevepiercy** [comment]
Please close https://github.com/plone/6.demo.plone.org/issues/3 when merged.

File: src/plone/volto/default_homepage/demo.py [hunk 1]
```diff
@@ -349,271 +349,538 @@
...
                                     },
-                                    {"text": " web site."},
+                                    {
+                                        "text": ""
+                                    }
                                 ],
+                                "type": "li"
                             },
+                            {
+                                "children": [
```
Comment (ksuess) [inline src/plone/volto/default_homepage/demo.py:559]: I'm not sure which list to provide here, maybe both: awesome Plone add-ons(backend) and also awesome Volto add-ons (https://github.com/collective/awesome-volto#awesome-volto). What do you think? If both lists, than maybe a short explanation would be helpful.

**Comment by ksuess** [comment]
This PR is definitly an enhancement for a new Volto project set up of an interested person willing to create a website with Plone Volto. 
Maybe I am wrong, but I think this PR is not affecting the demo https://6.demo.plone.org/ 
But surprise https://6.demo.plone.org/ also shows exactly this PR changes. Magic!

File: src/plone/volto/default_homepage/demo.py (hunk 1 above)
Comment (stevepiercy) [reply src/plone/volto/default_homepage/demo.py:559]: Fixed in 2dc3c7a90e4574323259d9792b1ea0c81704c7e5

**Comment by stevepiercy** [comment]
> But surprise https://6.demo.plone.org/ also shows exactly this PR changes. Magic!

No, that's me editing the site, per @sneridagh's instructions from https://github.com/plone/6.demo.plone.org/issues/3#issuecomment-1227083416. There is no way I was going to edit this file by hand!

**Review by ksuess — APPROVED** [review]

File: src/plone/volto/default_homepage/demo.py [hunk 2]
```diff
@@ -349,271 +349,560 @@
         "title": "Welcome to Plone 6!",
         "description": "Congratulations! You have successfully installed Plone.",
```
Comment (davisagli) [inline src/plone/volto/default_homepage/demo.py:350]: @stevepiercy I would change the description too; it doesn't make sense for a demo site

**Comment by davisagli** [comment]
Looks great! I would consider putting the "Demo" section with login info at the top of the page (moving from immediate action to links to gather more info) but that's probably just a matter of opinion and I don't feel strongly.

**Pull request merged**